langchain4j-azure-open-ai/src/test/java/dev/langchain4j/model/azure/AzureOpenAIResponsibleAIIT.java

```diff
@@ -38,7 +39,7 @@ void chat_message_should_trigger_content_filter_for_violence(String deploymentNa
 
         ChatLanguageModel model = AzureOpenAiChatModel.builder()
                 .endpoint(System.getenv("AZURE_OPENAI_ENDPOINT"))
-                .apiKey(System.getenv("AZURE_OPENAI_KEY"))
+                .tokenCredential(new DefaultAzureCredentialBuilder().build())
```

Comment:
I am wondering: should this be a default value in the builder?

Reply:
Do you mean we could have an `.tokenCredential()` method that takes no parameter, and in that case it would automatically do a `.tokenCredential(new DefaultAzureCredentialBuilder().build())`?

Reply:
No, I meant no need to call `.tokenCredential()` at all, this could be a default?

```java
ChatLanguageModel model = AzureOpenAiChatModel.builder()
                .endpoint(System.getenv("AZURE_OPENAI_ENDPOINT"))
                .deploymentName("gpt-4o")
                .build()
```

In this case `new DefaultAzureCredentialBuilder().build()` will be used in the builder by default.

Reply:
I wouldn't do this for 2 reasons:
- most people will use the API Key, so this default method won't work
- It requires the optional `com.azure:azure-identity` dependency to the project, the default method won't work by default...